Assets/MixedRealityToolkit.SDK/Experimental/FollowSolver/Follow.cs

```diff
@@ -409,10 +439,14 @@ private bool AngularClamp(Vector3 refPosition, Quaternion refRotation, Vector3 c
 
             Vector3 currentRefForward = refRotation * Vector3.forward;
             Vector3 refRight = refRotation * Vector3.right;
-            Vector3 refUp = refRotation * Vector3.up;
 
             bool angularClamped = false;
 
+            // Calculate the bounds to use with angular clamping. 
+            Bounds bounds;
+            GetBounds(gameObject, angularClampMode, out bounds);
```

Comment:
Is it possible that this call causes slowdown when using `ColliderBounds` and a complex `MeshCollider`? Might we be able to cache the results of `GetBounds` so we're not calling it every time we call `AngularClamp`. There's no need if this has already been tested and you don't think it's a significant problem.

Reply:
This is a good thought, initially I was thinking "this must be done every frame" in case the target content is changing in size. But, that seems more of the exception rather than the rule. So, I took your advice and it's now cached (with a public method to recalculate it if the user does have content changing in size).

Before:
![preporfile](https://user-images.githubusercontent.com/13305729/75801486-11670900-5d30-11ea-818a-7c8ba5678728.PNG)

After:
![postporfile](https://user-images.githubusercontent.com/13305729/75801457-044a1a00-5d30-11ea-9323-a57f1807f8b4.PNG)